doc/website/getting-started/installation.md

```diff
@@ -6,9 +6,10 @@ iceoryx_utils and iceoryx_posh are deployed as independent cmake packages. Posh
 
 ### Dependencies
 
+ - 64-bit hardware (e.g. x86_64 or aarch64)
```

Comment:
Here I would be a little bit more specific. Could you please mention that 32-bit hardware should be working in general but we only support 64-bit hardware.